fern/pages/sdks/getting-started/preview-your-sdk-locally.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: Preview your SDK
+description: Use Fern's CLI tool to preview your SDK locally. 
+---
+
+[Once you configure your SDK](/learn/sdks/getting-started/generate-your-first-sdk), you can preview the generated SDK code using Fern's CLI. 
+
+Simply append the `--preview` flag to the command used to generate the SDK and you will see the generated code populated in a `.preview` folder within your `fern` folder. 
+
+Here's an example of how you can preview your SDK: 
+
+<Steps>
+### Generator configuration
+```yaml generators.yml
+api: 
+  path: ./path/to/openapi.yml
+groups:
+  public:
+    generators:
+      - name: fernapi/fern-python-sdk
+        version: 3.0.0
+        output:
+          location: pypi
+          package-name: imdb
+          token: ${PYPI_TOKEN}
+        github:
+          repository: imdb/imdb-python
+        config:
+          client_class_name: imdb
+```
+
+### Invoke the Fern CLI
+
+```shell 
+fern generate --group public --preview
+```
+
+### Preview your SDK
+
+The resulting folder structure will look like this: 
+
+<Tabs>
+<Tab title="OpenAPI">
+```shell {3-5}
+fern/
+  ├─ fern.config.json
+  ├─ .preview/
+    └─ fern-python-sdk/
+      └─ ...
+  ├─ generators.yml
+  └─ openapi/
+    └─ openapi.yml
+```
+</Tab>
+<Tab title="Fern Definition">
+```shell {3-5}
+fern/
+  ├─ fern.config.json
+  ├─ .preview/
+    └─ fern-python-sdk/
+      └─ ...
+  ├─ generators.yml
+  └─ definition/
+    ├─ api.yml
+    └─ imdb.yml
+```
+</Tab>
+</Tabs>
```

Comment:
We should add a callout that explains that if you have made any manual modifications in the github repo of the SDK then `--preview` will also preserve those